federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -42,9 +42,9 @@ spec:
           mountPath: /srv/kubernetes/
           readOnly: true
       - name: etcd
-        image: quay.io/coreos/etcd:v2.3.3
+        image: gcr.io/google_containers/etcd:3.0.14-alpha.1
         command:
-          - /etcd
+          - /usr/local/bin/etcd
```

Comment:
So @madhusudancs, what is your suggestion for federation, we should make etcd3 as default, but still provide support for etcd2 or we should support etcd2 as default but optionally support etcd3?